scripts/build.js

```diff
@@ -21,7 +21,7 @@ const ChromeExtensionUtils = require('chrome-extension-utils');
 const Shared = require('./shared');
 const WebpackConfig = require('../webpack.config');
 
-const manifest = Shared.getManifest();
+const manifest = Shared.getManifest(process.env.NODE_ENV === 'production');
```

Comment:
I think it's best to isolated usage of any globals to a single file (entry point), it's more explicit (and also could re-use `isProd` variable in `webpack.config.js`)